apps/admin-gui/src/app/facilities/pages/facility-detail-page/facility-service-config/facility-service-config.component.ts

```diff
@@ -68,17 +68,14 @@ export class FacilityServiceConfigComponent implements OnInit {
   allowedStatuses: string[] =  ['INVALID', 'VALID'];
 
   ngOnInit() {
-    this.route.parent.params.subscribe(parentParams => {
-      const facilityId = parentParams['facilityId'];
+    this.facility = this.entityStorageService.getEntity();
+    this.facilityManager.getFacilityById(this.facility.id).subscribe(facility => {
+      this.facility = facility;
 
-      this.facilityManager.getFacilityById(facilityId).subscribe(facility => {
-        this.facility = facility;
+      this.facilityManager.getAssignedResourcesForFacility(facility.id)
+        .subscribe(resources => this.resources = resources);
 
-        this.facilityManager.getAssignedResourcesForFacility(facility.id)
-          .subscribe(resources => this.resources = resources);
-
-        this.serviceManager.getAssignedServices(facility.id).subscribe(services => this.services = services);
-      });
+      this.serviceManager.getAssignedServices(facility.id).subscribe(services => this.services = services);
     });
```

Comment:
The `facility` object is passed down and requires the name. We could store the name but there may be data freshness issue with that (name change). This service only stores ids since they are immutable. Also this page is not used anywhere as of now so maybe when it is used/reworked it will turn out this is not needed.